packages/ui/sass/components/chart.scss

```diff
@@ -0,0 +1,53 @@
+%counter-shared {
```

Comment:
Since the other components use the plural of the name, let's rename this one to `charts.scss` instead of `chart.scss`

Additionally, you still need to import your charts.scss file in the ./main.scss file else the file won't be accessible.

```
@import 'sass/components/charts';
```